src/OpenSage.Core/DeltaTimer.cs

```diff
@@ -76,5 +76,13 @@ public TimeInterval(TimeSpan totalTime, TimeSpan deltaTime)
     public static TimeInterval Zero { get; } = new TimeInterval();
 
     // This is the ratio between stopwatch ticks and timespan ticks
+
     private static readonly double TickRatio = 10000000.0 / Stopwatch.Frequency;
+
+    private const double MsPerLogicFrame = 1000.0 / 30.0;
```

Comment:
I think this might already be hardcoded somewhere else - can we reuse that instead of hardcoding the logic frames per second in multiple places?

https://github.com/OpenSAGE/OpenSAGE/blob/a3cef36b59cd9b83065a6f4da8d6f19d1b29ff83/src/OpenSage.Game/Game.cs#L55

Reply:
We should have this once in a central location, yes. We can't use `Game`'s constant in `OpenSage.Core` though, so maybe the constant should live somewhere in the core library.

Reply:
I think it does depend on the game though, since from the comment BFME's logic frame tick rate was 5hz rather than 30. Not sure of the best way to solve that.

Reply:
Why does this constant (and `LogicFrameRelativeDeltaTime `) need to be in `OpenSage.Core`? Could `LogicFrameRelativeDeltaTime ` instead be an extension method on `DeltaTimer` that's defined in `OpenSage.Game`?

Reply:
Good idea! Shame that C# (still) doesn't support extension properties, but a method is fine too.